entries/a/alaskaair.com.json

```diff
@@ -8,6 +8,14 @@
     },
     "keywords": [
       "transport"
+    ],
+    "regions": [
+      "ca",
+      "cu",
```

Comment:
```suggestion
```
Alaska Airlines no longer fly to Cuba - see https://newsroom.alaskaair.com/2017-11-14-Alaska-Airlines-will-discontinue-flying-to-Havana-Cuba